### PR TITLE
Sbt plugin

### PR DESCRIPTION
**Pull request by scelestino**
Work in Progress

### CODE REVIEW

**Comment by hugo-vrijswijk** [comment]
Very cool! Thanks a lot for your work! I will see if I can do anything for it tomorrow

File: sbt/src/main/scala/stryker4s/sbt/SbtReporter.scala [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package stryker4s.sbt
+
+import stryker4s.model.MutantRunResults
+import stryker4s.run.report.MutantRunReporter
+
+class SbtReporter extends MutantRunReporter {
+
+  // TODO: report info through sbt console
+  override def report(runResults: MutantRunResults): Unit = {
```
[the remark this answers is not in the document]
Comment (scelestino) [reply sbt/src/main/scala/stryker4s/sbt/SbtReporter.scala]: SBT has it own mechanism for logging called streams. It is the proper way to do that.

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,2 +1,9 @@
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.1")
+
+/**
+  * It can be replaced by enabling SbtPlugin since sbt 1.2.0
```
[the remark this answers is not in the document]
Comment (scelestino) [reply project/plugins.sbt]: Ok, and I going to add support to SBT 0.13.X (and Scala 2.11)

**Comment by hugo-vrijswijk** [comment]
I will merge this into the `sbt-plugin branch so I can have a go at it. Should you make more changes I think it's easiest to make a second PR

**Comment by scelestino** [comment]
@hugo-vrijswijk ok. This week I will be busy but I'm going to continue it next weekend.

**Pull request merged**